src/validators/drugorderedtreatments.js

```diff
@@ -76,6 +76,18 @@ export class DrugOrderedTreatmentValidator {
   }
 
   isValid() {
-    return validGenericTextfield(this.explanation) && this.validActionTaken()
+    return validGenericTextfield(this.explanation) && this.validActionTaken() && this.validOrderedBy()
+  }
+
+  validOrderedBy() {
+    if (this.orderedBy.length <= 1) {
+      return true
+    }
+    else {
+      if (this.orderedBy.includes("None")) {
```

Comment:
Given the code in the component, will this execute? It seems like the list of options can either have multiple selections that are not 'None', or 'None'

Reply:
Yes that right, looking back at it that else block won't run